syntax = "proto3";

package ni.pythonpanel.v1;

option cc_enable_arenas = true;
option csharp_namespace = "NationalInstruments.PythonPanel.V1";
option go_package = "pythonpanelv1";
option java_multiple_files = true;
option java_outer_classname = "PythonPanelServiceProto";
option java_package = "com.ni.pythonpanel.v1";
option objc_class_prefix = "NIPP";
option php_namespace = "NI\\PythonPanel\\V1";
option ruby_package = "NI::PythonPanel::V1";

message BoolCollection {
  repeated bool values = 1;
}

message ByteStringCollection {
  repeated bytes values = 1;
}

message FloatCollection {
  repeated double values = 1;
}

message IntCollection {
  repeated sint64 values = 1;
}

message StringCollection {
  repeated string values = 1;
}
